### PR TITLE
[ Ent-corda ] Adding nmap service

### PR DESCRIPTION
**Pull request by abevers**
**Changelog**
- Fix getting `nodeInfoFile` & `network-parameters-initial.conf` from Vault
- Update `notary-initial-registration` to have static `nodeInfoFile` name

**To be reviewed by**
@suvajit-sarkar 

**Linked issue**
#778

### CODE REVIEW

**Comment by suvajit-sarkar** [comment]
Looks fine to me

File: platforms/corda-ent/charts/nmap/templates/deployment.yaml [hunk 1]
```diff
@@ -17,7 +17,6 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      app: {{ .Values.nodeName }}
```
Comment (sownak) [inline platforms/corda-ent/charts/nmap/templates/deployment.yaml:20]: matchLabels must match metadata.labels

File: platforms/corda-ent/charts/nmap/templates/deployment.yaml [hunk 2]
```diff
@@ -85,13 +82,11 @@ spec:
                     sleep {{ $.Values.healthcheck.readinesscheckinterval }}
                   else
                    initial_param=$(echo ${LOOKUP_SECRET_RESPONSE} | jq -r '.data["network-parameters-initial"]')
-                   echo "${initial_param}" | base64 -d > ${MOUNT_PATH}/notary-nodeinfo/network-parameters-initial.conf
+                   echo "${initial_param}" | base64 -d > ${NODEINFO_MOUNT_PATH}/network-parameters-initial.conf
                    echo "Successfully got initial network parameters"
                    break
                 fi
                 COUNTER=`expr "$COUNTER" + 1`
-              done
```
Comment (jagpreetsinghsasan) [inline platforms/corda-ent/charts/nmap/templates/deployment.yaml:93]: Please add 'done' back again. Otherwise chart wont run

Comment (abevers) [reply platforms/corda-ent/charts/nmap/templates/deployment.yaml:93]: Fixed in commit [7568f6c](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/844/commits/7568f6cb9b32e8f50b7d860d97e999f40da288eb)

File: platforms/corda-ent/charts/nmap/templates/deployment.yaml (hunk 1 above)
Comment (abevers) [reply platforms/corda-ent/charts/nmap/templates/deployment.yaml]: Fixed in commit [7568f6c](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/844/commits/7568f6cb9b32e8f50b7d860d97e999f40da288eb)

**Comment by jagpreetsinghsasan** [comment]
Code seems fine.

**Pull request merged**